rules/integrations/google_workspace/collection_google_workspace_custom_gmail_route_created_or_modified.toml

```diff
@@ -27,7 +27,47 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Google Workspace Custom Gmail Route Created or Modified"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Google Workspace Custom Gmail Route Created or Modified
+
+Gmail is a popular cloud-based email service developed and managed by Google. Gmail is one of many services available for users with Google Workspace accounts.
+
+Threat actors often send phishing emails to corporate Gmail accounts containing malicious URL links or attachments. Google Workspace identity relies on the corporate user Gmail account and thus if stolen, allows threat actors to further intrusion efforts from valid user accounts.
```

Comment:
```suggestion
Threat actors often send phishing emails containing malicious URL links or attachments to corporate Gmail accounts. Google Workspace identity relies on the corporate user Gmail account and if stolen, allows threat actors to further intrusion efforts from valid user accounts.
```

Reply:
The second half of the last sentence needs to be revised. Do you mean that stolen Google Workspace identities allow threat actors to intrude on other valid user accounts?